Source/Core/PltDeviceHost.cpp

```diff
@@ -206,6 +206,14 @@ PLT_DeviceHost::Start(PLT_SsdpListenTask* task)
     NPT_TimeInterval repeat;
     repeat.SetSeconds(leaseTime?(int)((leaseTime >> 1) - 10):30);
 
+    /**
+     * M.Schenk 2010.11.26
+     * Songbook fix for device discovery
+     */
+#if 1
+    repeat.SetSeconds(5);
```

Comment:
You should call `SetLeaseTime(5)` instead if you need to do something non standard and you want to support a non compliant software like Songbook.